### PR TITLE
Gives survivors more stuff (and other survivor things)

### PR DESCRIPTION
**Pull request by Technical-l**
## About The Pull Request

Re-does all existing survivor jobs. 

All survivors now get a flashlight, radio, crowbar, and job specific gear (some of it is silly, can remove if needed). 

Added a new armor for the security officer job, same slowdown and coverage as light armor but much lower stats

Added a sledgehammer for the miner class, similar to a vali claymore but without the vali component

Rambo survivor no longer has rugged gear, now has heavily increased skills, some basic armor, and lots of gear. Renamed to 
overpowered survivor as it's quite strong for a survivor now.

Survivor doctors and scientists now spawn with labcoats, labcoats now have 25 acid armor.

Increased the damage of telescopic batons to 20, and the damage of butchers knives to 35

Other minor changes too

## Why It's Good For The Game

Working on the holy grail - survivors in TGMC. Thought I'd PR this for admin events and whatnot.
Xeno buffs make it harder for survivors to get off their feet, which this PR might alleviate, and it makes sense for people to still have their job specific gear after a xeno disaster breaks out on colony.

## Changelog

:cl: Joe13413
add: New survivor armor and melee weapon
balance: Survivors now spawn with job specific gear, radios, flashlights, and crowbars.
/:cl:

### CODE REVIEW

**Comment by Lauwma** [comment]
Based man

**Comment by Technical-l** [comment]
le epic merge conflict hold on real quick

**Comment by Technical-l** [comment]
image

**Comment by SimplySmiley123** [comment]
Finally, boon for survs..

**Comment by Technical-l** [comment]
closing because of #16401, i accidentally made a PR with everything this has but more

**Comment by Technical-l** [comment]
we are so back

**Comment by Technical-l** [comment]
> Shows images please, giga based pr btw

dont got access to my main pc right now but heres a fella wearing and wielding the new shit
image

File: code/datums/jobs/job/survivor.dm [hunk 1]
```diff
@@ -252,10 +396,74 @@ Good luck, but do not expect to survive."})
...
+/datum/outfit/job/survivor/salesman/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	. = ..()
+	H.equip_to_slot_or_del(new /obj/item/weapon/gun/pistol/holdout, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/pistol/holdout, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/pistol/holdout, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/pistol/holdout, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/tool/lighter/zippo, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/tool/crowbar, SLOT_IN_BACKPACK)
+
+	H.equip_to_slot_or_del(new /obj/item/spacecash/c500, SLOT_IN_BELT)
```
Comment (ivanmixo) [inline code/datums/jobs/job/survivor.dm:417]: Jeez, please just do a for loop

File: code/datums/jobs/job/survivor.dm [hunk 2]
```diff
@@ -273,20 +481,39 @@ Good luck, but do not expect to survive."})
...
+	H.equip_to_slot_or_del(new /obj/item/restraints/handcuffs, SLOT_IN_BELT)
+	H.equip_to_slot_or_del(new /obj/item/stack/medical/heal_pack/gauze, SLOT_IN_BELT)
+	H.equip_to_slot_or_del(new /obj/item/tool/crowbar, SLOT_IN_BELT)
 
 
-// Rambo Survivor
+// Rambo Survivor - pretty overpowered, pls spawn with caution
 /datum/job/survivor/rambo
-	title = "Survivor"
+	title = "Overpowered Survivor"
```
Comment (ivanmixo) [inline code/datums/jobs/job/survivor.dm:500]: Could you just stick with "Rambo Survivor" instead

**Review by ivanmixo — CHANGES_REQUESTED** [review]

File: code/datums/jobs/job/survivor.dm (hunk 1 above)
Comment (Technical-l) [reply code/datums/jobs/job/survivor.dm:417]: thinking back, its a stupid meme so i'm gonna get rid of the cash altogether

File: code/datums/jobs/job/survivor.dm (hunk 2 above)
Comment (Technical-l) [reply code/datums/jobs/job/survivor.dm]: sure

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**